woeip/templates/includes/_head.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Although this project probably won't be translated into other languages and locales, it's still a good practice to maintain good i18n/l10n habits:

```django
{% load i18n %}
{% get_current_language as LANGUAGE_CODE %}
<!DOCTYPE html>
<html lang="{{ LANGUAGE_CODE }}">
```